Added new docker compose deployment method and associated .env, and modified README

HrBingR opened the pull request:
I added a new way to deploy an almost ready to go docker compose, with an associated environment variable file (.env), and I added some short instructions on this to the readme.md

This should greatly simplify deployment for anyone trying to deploy Blink, as all environment variables have definitions on what they do as well.

JaneJeon commented:
One minor thing before I merge: does this setup work for you (i.e. can you stand up a working instance w/ the provided docker-compose.yml)? @HrBingR 

Thanks

JaneJeon commented:
@HrBingR oh, and it seems CI is failing due to linting issues. Could you run a `prettier --write .`?

HrBingR commented:
So some modifications are needed by the end user (I blanked out the HOST_URL and BASE_URL fields, and used generic fields for the database connection) but yes, I tested it both on localhost and functionally using real URLs through my reverse proxy (traefik).

Some instructions have been added to the docker-compose.yml and .env files where values are needed.

The required changes were also briefly noted in the new section I added to the README.md.

I have added one more commit to this pull request since my original submission, I removed the labels: field from the docker-compose.yml that was left in after my testing with traefik, and I've just re-tested on my instance with localhost and can confirm the above, everything works as intended.
